docs/05-technical-reference/00-custom-resources/svls-01-function.md

```diff
@@ -113,7 +113,7 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.buildResources.limits.memory**         |       No       | Defines the maximum amount of memory available for the Job's Pod to use.      |
 | **spec.buildResources.requests.cpu**          |       No       | Specifies the number of CPUs requested by the build Job's Pod to operate.       |
 | **spec.buildResources.requests.memory**       |       No       | Specifies the amount of memory requested by the build Job's Pod to operate.               |
-| **spec.runtime**                         |       No       | Specifies the runtime of the Function. The available values are `nodejs14` and `python39`. It is set to `nodejs14` unless specified otherwise.  |
+| **spec.runtime**                         |       No       | Specifies the runtime of the Function. The available values are `nodejs14`, `nodejs16` and `python39`. It is set to `nodejs14` unless specified otherwise.  |
```

Comment:
```suggestion
| **spec.runtime**                         |       No       | Specifies the runtime of the Function. The available values are `nodejs14`, `nodejs16`, and `python39`. It is set to `nodejs14` unless specified otherwise.  |
```

We use[ the Oxford comma (serial comma)](https://en.wikipedia.org/wiki/Serial_comma) in our documentation. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#punctuation) for reference.